packages/rum-core/.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
can you remove this file as its unrelated to the PR? We have our prettier rules defined here https://github.com/elastic/apm-agent-rum-js/blob/7b02f14f626e41eb14172064afb4d51076ee379a/.eslintrc.js#L40

Reply:
It's added by mistake.
I've already removed it.